jest.init.js

```diff
@@ -6,7 +6,7 @@
 
 import { config } from "@vue/test-utils";
 import Toast from "vue-toastification";
-import Tooltip from "primevue/tooltip";
+import VueTippy from "vue-tippy";
```

Comment:
we can use a more readable name?
```suggestion
import VueTooltip from "vue-tippy";
```